### PR TITLE
Add method to check if it is a mongo manipulator

### PR DESCRIPTION
**Pull request by dstiliadis**
used for testing.

### CODE REVIEW

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pipeline",
    "component": "manipulate",
    "pr-id": "154",
    "commit-sha": "231c3e2c2f4b454912f99b6dc521c20d3b450113",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "underwater",
    "pr-id": "154",
    "commit-sha": "e332ac36e0d61bb792d33dd765a3db7331c4842c",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "backend",
    "pr-id": "1270",
    "commit-sha": "a478d9edfab777427d0b5f62107d219b0a0bd699",
    "pipeline": "master"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pipeline",
    "component": "manipulate",
    "pr-id": "154",
    "commit-sha": "231c3e2c2f4b454912f99b6dc521c20d3b450113",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "underwater",
    "pr-id": "154",
    "commit-sha": "e332ac36e0d61bb792d33dd765a3db7331c4842c",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "backend",
    "pr-id": "1270",
    "commit-sha": "694076d50704f65c751174ddb9c4bd5eca2cdc14",
    "pipeline": "master"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pipeline",
    "component": "manipulate",
    "pr-id": "154",
    "commit-sha": "231c3e2c2f4b454912f99b6dc521c20d3b450113",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "underwater",
    "pr-id": "154",
    "commit-sha": "21b737f97a6e02b367aaec0d22ec2069ab4e99b0",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "backend",
    "pr-id": "1270",
    "commit-sha": "694076d50704f65c751174ddb9c4bd5eca2cdc14",
    "pipeline": "master"
  }
]
```

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pipeline",
    "component": "manipulate",
    "pr-id": "154",
    "commit-sha": "231c3e2c2f4b454912f99b6dc521c20d3b450113",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "underwater",
    "pr-id": "154",
    "commit-sha": "6494d243fe8e9692fcbcdc806eadd68dc02c19c0",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "backend",
    "pr-id": "1270",
    "commit-sha": "694076d50704f65c751174ddb9c4bd5eca2cdc14",
    "pipeline": "master"
  }
]
```

File: manipmongo/helpers.go [hunk 1]
```diff
@@ -283,3 +283,10 @@ func IsUpsert(mctx manipulate.Context) bool {
 	_, upsert := mctx.(opaquer).Opaque()[opaqueKeyUpsert]
 	return upsert
 }
+
+// IsMongoManipulator returns true if this is a mongo manipulator
+func IsMongoManipulator(manipulator manipulate.Manipulator) bool {
+	_, ok := manipulator.(*mongoManipulator)
+
+	return ok
+}
```
Comment (t00f) [inline manipmongo/helpers.go:292]: Does it really worth it to have it here?

**Review by t00f — COMMENTED** [review]
Not sure if it's necessary, we can just add the check in underwater, no ?

**Comment by dstiliadis** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pipeline",
    "component": "manipulate",
    "pr-id": "154",
    "commit-sha": "231c3e2c2f4b454912f99b6dc521c20d3b450113",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "underwater",
    "pr-id": "154",
    "commit-sha": "6494d243fe8e9692fcbcdc806eadd68dc02c19c0",
    "pipeline": "master"
  },
  {
    "project": "pipeline",
    "component": "backend",
    "pr-id": "1270",
    "commit-sha": "f1a087b37cd9388eb3be930de7c33209fabaeb06",
    "pipeline": "master"
  }
]
```

**Review by t00f — APPROVED** [review]
Ok `mongoManipulator` is a private structure.

**Pull request merged**